packages/react/src/components/Toggle/index.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+import { safeInvoke } from "../../utils";
+
+const Toggle = ({ checked, disabled, label, helper, onChange }) => {
+  const [isOn, setIsOn] = useState(checked);
+
+  const handleToggle = () => {
+    const newState = !isOn;
+    setIsOn(newState);
+    safeInvoke(onChange(newState));
+  };
+
+  return (
+    <div
+      className={cx("ToggleContainer", {
+        "is-on": isOn,
+        "is-disabled": disabled,
+      })}
+    >
+      <div className="Toggle" onClick={!disabled && handleToggle}>
+        <div className="Toggle-bullet" />
+        <span className="Toggle-bullet-label is-on">ON</span>
```

Comment:
```suggestion
        <span className="Toggle-bullet-label Toggle-bullet-label--on">ON</span>
```

The `is-` shouldn't be used here because it's not a state per se but rather a modifier 🙂 
It's no this item which is `is-on`, but the `ToggleContainer`

Reply:
👍